cmd/command/cd/cd_clusters.go

```diff
@@ -118,19 +118,12 @@ func (p *Plural) cdClusterCommands() []cli.Command {
 }
 
 func (p *Plural) handleListClusters(_ *cli.Context) error {
```

Comment:
This actually does work, but we can also use the `consoleInstances` query, which just selects Plural cloud clusters.

Reply:
I searched the whole `schema.graphql` and couldn't find this query